internal/operators/ocs/ocs_operator.go

```diff
@@ -133,11 +132,12 @@ func (o *operator) GetHostRequirements(_ context.Context, cluster *common.Cluste
 		return &models.ClusterHostRequirementsDetails{CPUCores: 0, RAMMib: 0}, nil
 	}
 
+	// worker and auto-assign
 	if disks > 0 {
 		// for each disk ocs requires 2 CPUs and 5 GB RAM
 		return &models.ClusterHostRequirementsDetails{
-			CPUCores:   CPUMinimalMode + disks*o.config.OCSRequiredDiskCPUCount,
-			RAMMib:     conversions.GbToMib(MemoryGBMinimalMode + disks*o.config.OCSRequiredDiskRAMGB),
+			CPUCores:   CPUMinimalMode + reqDisks*o.config.OCSRequiredDiskCPUCount,
```

Comment:
I think it doesn't need to be changed after all, as in this case `disks` will always be equal to `reqDisks`.

Reply:
Done